service/lib/agama/manager.rb

```diff
@@ -144,6 +144,7 @@ def install_phase
       logger.error "Installation error: #{e.inspect}. Backtrace: #{e.backtrace}"
     ensure
       service_status.idle
+      installation_phase.finish
```

Comment:
as it is in ensure it means that even if installation failed, it will be in finish phase. Is it correct?

Reply:
Yes. It does not change the behavior at all.

Reply:
It would be great to have some kind of error summary if something goes wrong.